server/src/Metrics.hs

```diff
@@ -31,6 +31,8 @@ data IcepeakMetrics = IcepeakMetrics
   , icepeakMetricsQueueAdded        :: Counter
   , icepeakMetricsQueueRemoved      :: Counter
   , icepeakMetricsSyncDuration      :: Histogram
+  , icepeakMetricsWsQueueAdded      :: Counter
+  , icepeakMetricsWsQueueRemoved    :: Counter
```

Comment:
Shall we just add the counter metric for skipped updates in this PR, too?

Reply:
Good thinking, will add!